_posts/elements/2015-05-12-forms.md

```diff
@@ -0,0 +1,284 @@
+---
+layout:       element
+title:        Forms
+date:         2015-05-12 16:02:35
+
+categories:   elements
+tags:
+- atomic
+- pattern
+- forms
+
+slug:         forms
+scssurl:      "https://github.com/edx/ux-pattern-library/blob/master/_src/edx-pl/sass/components/_forms.scss"
+
+description:  Various examples of different form fields and their usage and options.
+
+info: Forms allow users to interact with the interface, set preferences, and enter any necessary personal information. They also form the base of individual exercises within each class.
+---
+
+<h3 class="hd-4 example-set-hd">Forms</h3>
+<h4 class="hd-6">Text inputs</h4>
+<div class="example-set">
+    <fieldset class="fieldset">
+        <legend class="group-legend is-hidden">Personal Details</legend>
+        <div class="field-wrapper">
+            <label class="form-label" for="full_name_default">Your name (default text field)</label>
+            <input class="input input-text" type="text" name="full_name_default" id="full_name_default" placeholder="e.g. Jane Doe" required>
```

Comment:
My OCD is kicking in, these two sibling elements have different naming conventions - ``form-label`` and ``input``. Please sync these up to either be ``form-label`` + ``form-input`` or ``label`` + ``input``. The former set is more scoped to form elements, so that may be a plus.